danger-kotlin-library/dependencies.gradle

```diff
@@ -3,6 +3,6 @@ apply from: file('../dependencyVersions.gradle')
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     utilsDependencies()
-    kotlinDependencies()
+    //kotlinDependencies()
```

Comment:
Commented code :P

Reply:
yeah i'm not finish with my tests

Reply:
yeah i know :) not finished with my tests